distributed/protocol/serialize.py

```diff
@@ -765,12 +765,11 @@ def _serialize_array(obj):
 @dask_deserialize.register(array)
 def _deserialize_array(header, frames):
     a = array(header["typecode"])
-    for f in map(memoryview, frames):
-        try:
-            f = f.cast("B")
-        except TypeError:
-            f = f.tobytes()
-        a.frombytes(f)
```

Comment:
From this, do I read correctly that before if you tried deserializing an array from multiple frames it would *always* fail?
If so, how's it possible that we never noticed before? What's the threshold for serialize() to break up an array into multiple frames?